zfsbootmenu/lib/kmsg-log-lib.sh

```diff
@@ -84,3 +84,52 @@ zerror() {
   : > "${BASE}/have_errors"
   echo "<3>ZFSBootMenu: $1" > /dev/kmsg
 }
+
+# arg1: comma-separated log levels to print
+# prints: all logs at that level
+# returns: nothing
+
+print_kmsg_logs() {
+  local levels levels_array grep_args
+
+  levels="${1}"
+  if [ -z "${levels}" ]; then
+    zerror "levels is undefined"
+    return
+  fi
+
+  # Try to use feature flags found in dmesg from util-linux
+  if output="$( dmesg --notime -f user --color=always -l "${levels}" 2>/dev/null )" ; then
+    echo -e "${output}"
+  # fall back to busybox if we have it
+  elif [ -n "${HAS_BUSYBOX_DMESG}" ]; then
```

Comment:
Is this test even necessary? It already attempts to use util-linux, so maybe this can just be an all-catching `else`. Worse case, maybe just change to
```bash
elif output="$( dmesg -r )"
[...]
```
and grep off `$output` after the level parsing.